chain-signatures/node/src/indexer.rs

```diff
@@ -324,7 +328,7 @@ pub fn run(
                 anyhow::Ok(lake)
             }) else {
                 tracing::error!(?options, "indexer failed to build");
-                backoff(i);
+                backoff(i, 1, 120);
```

Comment:
Seems i will never decrease, and eventually i will become `max`  if indexer keep working. Do we need to restore it to a small number on a not failing case?